indra/llrender/llrendertarget.cpp

```diff
@@ -475,7 +475,7 @@ void LLRenderTarget::clear(U32 mask_in)
 
 U32 LLRenderTarget::getTexture(U32 attachment) const
 {
-    if (attachment > mTex.size()-1)
```

Comment:
What's the point of this change?

Reply:
> What's the point of this change?

It doesn't work in case of an empty container

Reply:
Right, unsigned int under the hood, so zero minus one won't work correctly and looks like it can be empty.

Reply:
But why does it crash?
Even if condition doesn't work, execution would check mTex.empty() and return zero, not crash on getTexture().

Reply:
Not sure how the optimizer transforms such ugly code